platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/StartupStateUtils.java

```diff
@@ -326,7 +329,12 @@ private static ReservedSignedState buildGenesisState(
             @NonNull final MerkleRoot stateRoot) {
 
         if (!configuration.getConfigData(AddressBookConfig.class).useRosterLifecycle()) {
-            initGenesisPlatformState(configuration, stateRoot.getWritablePlatformState(), addressBook, appVersion);
+            initGenesisState(
```

Comment:
Why remove the term `Platform`?   Leaving it generic makes it sound like initing global state.